src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    if (parseFloat(two) === 0) {
+      return '0';
+    }
```

Comment:
dont return zero, return an error on the display screen, something like "Division by 0 error" or some other creative error. Just look for a way to let user know it's an error.